.changeset/grumpy-toys-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+"@redocly/cli": patch
+---
+
+Removed descriptions adding for x-tagGroups for the `join` command.
```

Comment:
Please include something about why this change was made, and the differences that the user may observe.